.github/integration/tests/sda/60_api_admin_test.sh

```diff
@@ -66,6 +66,42 @@ if [ -z "$output" ] ; then
     echo "Uploaded file not in inbox"
     exit 1
 fi
+
+# download the file, re-encrypted with the client key
+clientPubKey="$(base64 -w0 /shared/client.pub.pem)"
+outFile="download_reenc_NA12878.bam.c4gh"
+resp="$(curl -s -k -L -w "%{http_code}\n" -H "Authorization: Bearer $token" -H "C4GH-Public-Key: $clientPubKey" "http://api:8080/users/test@dummy.org/file/$fileid" -o $outFile)"
```

Comment:
```suggestion
resp="$(curl -s -k -L -w "%{http_code}\n" -H "Authorization: Bearer $token" -H "C4GH-Public-Key: $clientPubKey" "http://api:8080/users/test@dummy.org/file/$fileid" -JO)"
```

This should use the `-J & -O` flags to ensure the content disposition name is used correctly.

Reply:
Use another file at the later stage then or use another file ID here.
This test should use the -J -Oflags to ensure that the file is named correctly when the user doesn't supply an output filename.

Reply:
The problem turned out to be the incorrect formatting of `Content-Disposition`. I should not just copied it from `sda-download` https://github.com/neicnordic/sensitive-data-archive/blob/main/sda-download/api/sda/sda.go#L338. Fixed in 4dc4317c and 8018ec4f